etl/src/main/java/org/pih/hivmigration/etl/sql/InfantMigrator.groovy

```diff
@@ -1,7 +1,5 @@
 package org.pih.hivmigration.etl.sql
 
-import org.w3c.dom.Text
```

Comment:
Thanks!

Reply:
IntelliJ does it automatically, didn't even notice this!